src/components/Room/Room.tsx

```diff
@@ -96,9 +113,24 @@ export function Room({
   }
 
   receiveMessage(message => {
+    const userSettings = settingsContext.getUserSettings()
+    !shellContext.tabHasFocus &&
+      userSettings.playSoundOnNewMessage &&
+      playNewMessageSound()
     setMessageLog([...messageLog, { ...message, timeReceived: Date.now() }])
   })
 
+  const playNewMessageSound = () => {
+    if (!audioBufferContainer.current) {
+      console.error("Can't fetch sound buffer to play")
```

Comment:
I don't know that this error message is quite accurate. This line may be hit if the audio data hasn't completed loading yet, but will eventually successfully load. What do you think of a message more like this?

```suggestion
      console.error('Audio buffer not available')
```